portfolio/src/main/webapp/script.js

```diff
@@ -63,4 +63,59 @@ function random_image() {
   // Remove the previous image.
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
+}
+
+
```

Comment:
Lots of extra white space here; please review the JavaScript style guide:
https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace